src/iocore/net/SSLUtils.cc

```diff
@@ -371,6 +372,7 @@ ssl_client_hello_callback(const SSL_CLIENT_HELLO *client_hello)
   return ssl_select_cert_success;
 }
 #endif
+#endif
```

Comment:
Hmm adding an #endif without adding additional #if ?  I wonder how this compiles.